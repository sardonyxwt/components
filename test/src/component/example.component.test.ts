import * as React from 'react';
import {
    createRenderTest,
    createSnapshotTest,
} from '@test/helper/testing.helper';
import { Example } from '@source';

describe('Example Component Test', () => {
    createRenderTest(
        'Example component default',
        React.createElement(Example, {}),
    );
    createSnapshotTest(
        'Example component default',
        React.createElement(Example, {}),
    );
    createSnapshotTest(
        'Example component with useReversColor',
        React.createElement(Example, {
            useReversColor: true,
        }),
    );
});
