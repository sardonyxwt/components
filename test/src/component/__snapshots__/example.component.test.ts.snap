// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Component Test Snapshot (Example component default) 1`] = `"<div class=\\"sc-AxjAm fPfvhw\\">Hello World!</div>"`;

exports[`Example Component Test Snapshot (Example component with useReversColor) 1`] = `"<div class=\\"sc-AxjAm iLNCPx\\">Hello World!</div>"`;
